Source/WebCore/workers/WorkerGlobalScope.cpp

```diff
@@ -82,6 +82,9 @@
 #include <wtf/WorkQueue.h>
 #include <wtf/threads/BinarySemaphore.h>
 
+#if ENABLE(WEBDRIVER_BIDI)
+#include "AutomationInstrumentation.h"
+#endif
```

Comment:
Nit: newline after #endif

Reply:
Thanks for the review. I just addressed Devin's comments, before actually merging the PR.